### PR TITLE
commands/exec: stop truncating >4095-byte commands

### PR DESCRIPTION
**Pull request by mstoeckl**
This commit fixes a rare edge case, in which very long commands are silently truncated. One way to test this is to compare the following two shell commands; if truncation occurs, the first will fail to set the background color.

```
swaymsg exec "swaynag -m `(head -c 4096 < /dev/zero | tr '\0' 'a')` --background 00FF00"

swaynag -m `(head -c 4096 < /dev/zero | tr '\0' 'a')` --background 00FF00
```

### CODE REVIEW

**Review by emersion — APPROVED** [review]
Thanks

File: sway/commands/exec_always.c [hunk 1]
```diff
@@ -36,17 +36,12 @@ struct cmd_results *cmd_exec_process(int argc, char **argv) {
 	}
 
 	if (argc == 1 && (argv[0][0] == '\'' || argv[0][0] == '"')) {
-		tmp = strdup(argv[0]);
-		strip_quotes(tmp);
+		cmd = strdup(argv[0]);
```
Comment (Mic92) [inline sway/commands/exec_always.c:39]: This code is missing a NULL check in case it goes out of memory.

Comment (emersion) [reply sway/commands/exec_always.c:39]: We don't have a OOM checking policy in Sway, yet. Many other places would need to get fixed.

Patches welcome I suppose.
